core/bentley/src/MobileUtils.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @internal */
+export class MobileUtils {

```

Comment:
Seems weird for `isITwinBackend/Frontend` to be defined on something called `MobileUtils`.

Reply:
these functions are used to tell if the current process is the frontend of an iTwin mobile app. They should return false for mobile browsers.